packages/keyring-api/src/rpc.ts

```diff
@@ -51,6 +51,7 @@ export enum KeyringRpcMethod {
   SubmitRequest = 'keyring_submitRequest',
   ApproveRequest = 'keyring_approveRequest',
   RejectRequest = 'keyring_rejectRequest',
+  SetSelectedAccounts = 'keyring_setSelectedAccounts',
```

Comment:
We need a `SetSelectedAccounts{Request,Response}Struct` structs and types (like for the other RPC requests in this file)